packages/ckeditor5/src/plugin.js

```diff
@@ -61,8 +61,10 @@ export default class MathType extends Plugin {
    * Inherited from Plugin class: Executed when CKEditor5 is destroyed
    */
   destroy() {
-    // eslint-disable-line class-methods-use-this
-    currentInstance.destroy();
+    if (currentInstance) {
+      // eslint-disable-line class-methods-use-this
+      currentInstance.destroy();
+    }
```

Comment:
Would this work? Just to keep it a little bit cleaner.

```suggestion
      // eslint-disable-line class-methods-use-this
      currentInstance?.destroy();
```

Reply:
Yes for sure! I just committed your suggestion 😄